src/main/flight/pid.c

```diff
@@ -55,7 +55,7 @@ int16_t axisPID[3];
 int32_t axisPID_P[3], axisPID_I[3], axisPID_D[3];
 #endif
 
-uint8_t dynP8[3], dynI8[3], dynD8[3];
+uint8_t dynP8[3], dynI8[3], dynD8[3], PIDscaler[3];
```

Comment:
I would add a comment here where the variable is declared that it is a scale factor for pitch and roll PIDs which is derived from the throttle, and 100 = 1.0x scale.